discoveryengine/documents_sample_test.py

```diff
@@ -15,17 +15,20 @@
 
 import os
 
-from discoveryengine import import_documents_sample
-from discoveryengine import list_documents_sample
-from discoveryengine import purge_documents_sample
-
 import pytest
 
+from discoveryengine import (
+    import_documents_sample,
+    list_documents_sample,
+    purge_documents_sample,
+)
+
 project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
 location = "global"
 data_store_id = "test-structured-data-engine"
 
 
+@pytest.mark.skip(reason="Table deleted.")
```

Comment:
This change is unrelated to the description of this PR. If you need to make this change, please create a separate PR and a related issue/bug if this is a temporary issue that needs to be tracked.

Reply:
Correct, the reason I added this is to prevent test failures from blocking the PR. A BQ table needed for this test to pass was deleted without my knowledge, so I added this skip in so the current PR can go through.